src/Compilers/CSharp/Portable/Binder/Binder_Statements.cs

```diff
@@ -844,6 +856,45 @@ protected BoundExpression BindInferredVariableInitializer(DiagnosticBag diagnost
             return expression;
         }
 
+        internal BoundExpression BindToNaturalType(BoundExpression expression, DiagnosticBag diagnostics)
```

Comment:
Why is this in `Binder_Statements.cs`? Seems like it should be in `Binder_Expressions.cs`

Reply:
+1
Also, consider adding xml doc

---
In reply to: [301341152](https://github.com/dotnet/roslyn/pull/37052#discussion_r301341152) [](ancestors = 301341152)